install_script.sh.template

```diff
@@ -1419,6 +1436,8 @@ elif [ ! "$no_agent" ]; then
   manage_security_and_system_probe_config "$sudo_cmd" "$security_agent_config_file" "$system_probe_config_file" "$DD_RUNTIME_SECURITY_CONFIG_ENABLED" "$DD_COMPLIANCE_CONFIG_ENABLED"
 fi
 
+manage_client_libraries_security_config "$sudo_cmd" "$environment_file" "$DD_APPSEC_ENABLED" "$DD_IAST_ENABLED" "$DD_APPSEC_SCA_ENABLED"
```

Comment:
Nit: two variables names start with `DD_APPSEC_` and one only with `DD_`. Is this intentional? Would it make more sense to make them all start with `DD_APPSEC_`?

Reply:
While I agree with your comments, these settings have be implemented and document in the client libraries for quite sometime, so are not possible to change without requesting all customers using them to migrate.